MAUI/Cartesian-Charts/Annotation.md

```diff
@@ -0,0 +1,743 @@
+
+The [ChartLabelStyle]() property allows you to customize the axis labels by changing the font family, font size, font attributes, and text color. It is the default style for the [AxisLabelStyle]() property. The following properties can be used to customize the axis label:
+
+* [TextColor]() - Changes the text color.
+* [BackgroundColor]() - Changes the background color of the text.
+* [Margin]() - Sets the margin for the text.
+* [Font]() - Changes the text size, font family, and font weight. (This is a deprecated API. Use FontSize, FontFamily, and FontAttributes properties instead.)
+* [FontFamily]() - Changes the font family for the text.
+* [FontAttributes]() - Changes the font style for the text.
+* [FontSize]() - Changes the font size for the text.
+
+## Adding arrow to vertical and horizontal line annotations
+
+To display a single-headed arrow, set the [LineCap]() property to [Arrow](). The default value of the [LineCap]() property is [None]().
+
+{% tabs %}
+
+ {% highlight xaml %}
+
+        <chart:SfCartesianChart>
+        ...
+        <chart:SfCartesianChart.Annotations>
+            <chart:VerticalLineAnnotation X1="2" LineCap="Arrow"/>
+            <chart:HorizontalLineAnnotation Y1="20" LineCap="Arrow"/>
+        </chart:SfCartesianChart.Annotations>
+
+        </chart:SfCartesianChart>
+
+{% endhighlight %} 
+
+{% highlight c# %}
+
+    SfCartesianChart chart = new SfCartesianChart();
+
+    var vertical = new VerticalLineAnnotation()
+    {
+        X1 = 2,
+        LineCap = ChartLineCap.Arrow
+    };
+
+    var horizontal = new HorizontalLineAnnotation()
+    {
+        Y1 = 20,
+        LineCap = ChartLineCap.Arrow
+    };
+
+    chart.Annotations.Add(vertical);
+    chart.Annotations.Add(horizontal);
+
+    this.Content = chart;
+
+{% endhighlight %}
+
+{% endtabs %}  
+
+![Vertical and horizontal Line Annotation with linecap in MAUI Chart](Annotation_image/vertical_horizontal_linecap.png)
+
+## Adding text in shape annotation
+
+For all shape annotations, the text can be displayed using the [Text]() property.
+
+{% tabs %}
+
+{% highlight xaml %}
+
+    <chart:SfCartesianChart>
+    ...
+        <chart:SfCartesianChart.Annotations>
+
+        <chart:EllipseAnnotation X1="2" X2="4" Y1="10" Y2="15" Text="Ellipse"/>
+        
+        </chart:SfCartesianChart.Annotations>
+    
+    </chart:SfCartesianChart>
+
+{% endhighlight %}
+
+{% highlight c# %}
+
+    SfCartesianChart chart = new SfCartesianChart();
+    ...
+    var ellipse = new EllipseAnnotation()
+    {
+        X1 = 2,
+        Y1 = 10,
+        X2 = 4,
+        Y2 = 15,
+        Text = "Ellipse"
+    };
+
+    chart.Annotations.Add(ellipse);
+
+    this.Content = chart;
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![Annotation in MAUI Chart](Annotation_image/ellipse_annotation.png)
+
+## Customizing text in shape annotation
```

Comment:
Updated the property description -https://help.syncfusion.com/maui/cartesian-charts/axis/axislabels#label-customization

Reply:
@SaiyathAliFathima , updated as like in maui